lib/cc/service/formatter.rb

```diff
@@ -19,7 +19,7 @@ def service_title
   end
 
   def message_prefix
-    prefix = options.fetch(:prefix, "").to_s
+    prefix = options.fetch(:prefix, "").to_s.dup
```

Comment:
I believe given the previously shared reasoning this is more appropriate 
```suggestion
    prefix = (options[:prefix] || "").to_s
```

Reply:
Fede, I remember trying this option too and both work as well, I'll take your suggestion 🙏